common/djangoapps/microsite_configuration/backends/__init__.py

```diff
@@ -0,0 +1 @@
+# Microsite backends shipped with openedx
```

Comment:
Can we include a little bit of information here about the different backends that are supported?